### PR TITLE
Improve docker compose

### PR DESCRIPTION
**Pull request by pjlampil**
- Refactored persister project
- Added PostgreSQL (timescaledb)
- Better docker-compose setup for development
- Pre-commit support for all of the projects

### CODE REVIEW

File: .env.dev-sample [hunk 1]
```diff
@@ -21,3 +23,10 @@ KAFKA_PASSWORD="strong422example.!_password,463qteuayrdfghfdg"
 ### Endpoint
 ALLOWED_IPS="*"  # Set "*" for all, otherwise comma separated list of IP addresses
 ENDPOINT_PATH="sensornode"  # Without leading slash
+
+### Postgres
+POSTGRES_DB=heldatapipe
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```
Comment (aapris) [inline .env.dev-sample]: Never use easily guessable password for any user even in docker

File: persister/persister/settings.py [hunk 2]
```diff
@@ -75,11 +75,21 @@
...
 # https://docs.djangoproject.com/en/3.1/ref/settings/#databases
 
+# DATABASES = {
+#     "default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}
+# }
+
 DATABASES = {
-    "default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
```
Comment (aapris) [inline persister/persister/settings.py:84]: Engine may be postgis in the future but this is good for now

**Review by aapris — CHANGES_REQUESTED** [review]
Use strong postgres password in .env.dev-sample

File: .env.dev-sample (hunk 1 above)
Comment (pjlampil) [reply .env.dev-sample]: This has been pretty much the standard approach for the development in all of the projects.

**Review by aapris — APPROVED** [review]
All good now :)

**Pull request merged**